src/views/states/list/components/utils.ts

```diff
@@ -15,6 +18,14 @@ export const interfaceFilters = {
     if (!selectedInput.length) return true;
     return selectedInput.some((ipType) => !!obj[ipType]);
   },
+  [FILTERS_TYPES.IP_ADDRESS]: (selectedInput, obj) => {
+    const searchIpAddress = selectedInput?.[0];
+    if (!searchIpAddress) return true;
+
+    const addresses = [obj?.ipv4?.address?.[0]?.ip, obj?.ipv6?.address?.[0]?.ip].filter(Boolean);
```

Comment:
nitpick: I suggest creating selectors for these. Something like `getIPv4Address` and `getIPv6Address`. If they're always used together like this, you could also create `getIPAddresses`.